src/main/java/service/strategy/strategyimpl/SupplyOperation.java

```diff
@@ -0,0 +1,13 @@
+package service.strategy.strategyimpl;
+
+import dao.Storage;
+import model.FruitRecord;
+
+public class SupplyOperation implements OperationHandler{
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.